test/passing/tests/repl.mli.ref

```diff
@@ -0,0 +1,39 @@
+(** Blocks such as this one:
+
+    {[
+      let x = 2 in
+      x + x
+    ]}
+
+    Or, as of odoc 2.1, this one:
+
+    {@ocaml[ let x = 2 ]}*)
```

Comment:
It's better if Mdx and OCamlformat agree on this ! (especially top-level blocks, which are re-written by Mdx)
These blocks are supposed to be big, so breaking the line is more appropriate I think.

Reply:
I agree it's better to put the block delimiters on their own line, I will even extract that in a separate PR because it should have been the case for master.
I diversified the tests to showcase what is allowed or not, but I still have to handle the "base indentation level" part of the spec.